errors.go

```diff
@@ -53,4 +53,9 @@ var (
 	ErrDigestUnknown = errors.New("could not compute digest of item")
 	// ErrLayerNotMounted is returned when the requested information can only be computed for a mounted layer, and the layer is not mounted.
 	ErrLayerNotMounted = errors.New("layer is not mounted")
+	// ErrNotSupported is returned when the requested functionality is not supported.
+	ErrNotSupported = errors.New("not supported")
```

Comment:
It doesn't hurt, but I'm not quite sure why you added this one?  It's not in the original patch, but is present in the code.  Do you need it for a future change up the chain?  I'm not seeing another reference to it in this PR.

Reply:
This PR was intended for `release-1.19` but we are not doing anything there instead we are using https://github.com/containers/storage/pull/1173 for `release-1.23` and `v3.4.2-rhel` therefore we might not need this PR I think we can revert this branch.